notifications_utils/clients/redis/redis_client.py

```diff
@@ -129,7 +129,7 @@ def exceeded_rate_limit(self, cache_key, limit, interval, raise_exception=False)
         else:
             return False
 
-    def add_key_to_sorted_set(self, cache_key, score, value, raise_exception=False):
+    def add_key_to_sorted_set(self, cache_key, sorted_set: dict, raise_exception=False):
```

Comment:
Let's either finish up the doc string here or remove it, It's pretty self explanatory, and add types to the params.